api_docs/markdowns/api/collections/collections.md

```diff
@@ -0,0 +1,227 @@
+# Collections API -- Getting Started [Alpha Version]
+
+## Summary
+
+The goal of this API is to give the possibility to cross data from different sources and different periods on a project. These data sources are named collections in this context and represent data coming from Botify Crawls, the Log Analyzer, Google Search Console and others.
+
+In the manner of an analytics tool, the collections API is queried by providing a set of dimensions and metrics to aggregate data.
+
+## Data Querying Endpoint
+
+- Operation: [[projectQuery;reference#/Project/projectQuery]]
+- Path: `projects/{username}/{project_slug}/query`
+- HTTP Verb: POST
+- Response: `Pagination<BQL_COLLECTIONS_RESULT>`
+
+Body:
+```JSON
+{
+  "collections": [COLLECTIONS],
+  "periods": [PERIODS],
+  "query": BQL_COLLECTIONS_QUERY
+}
+```
+
+### Collections
+
+A Botify Project has multiple collections usable for querying. To retrieve the available collections one can query the endpoint:
+
+- Operation: [[getProjectCollections;reference#/Collections/getProjectCollections]]
+- Path: `projects/{username}/{project_slug}/collections`
+- HTTP Verb: GET
+- Response: `List<Collection>`
+
+Three types of collections are to be distinguished:
+- Timestamped collections. Most collections are timestamped and represent a data source that evolves daily. This includes, among others, the Log Analyzer, Search Console and Visits features.
+- Non-timestamped collections. These collections represent a snapshot at a certain moment in time. This includes the Botify Crawl Reports.
+- "Global" collection. This unique collection is present on every project. It exposes dimensions that are available on all collections. This contains the URL Scheme and Segment data.
+
+The most frequently used collections are:
+- `crawl.YYYYMMDD`
+- `search_console`
+- `log_analyzer.google`
+
+### Collection details
+
+To get the available dimensions and metrics, a collection exposes the endpoint:
+- Operation: [[getCollectionDetail;reference#/Collections/getCollectionDetail]]
+- Path: `projects/{username}/{project_slug}/collections/{collection_id}`
+- HTTP Verb: GET
+- Response: `Datamodel`
+
+The `collection_id` is being retrieved through the collections list endpoint. It is also the name used for the collection on the data querying endpoint.
```

Comment:
I don't understand the second sentence, can you give an example?

Reply:
Hmm, examples are provided at the bottom of the page.
I changed the sentence a bit to make it clearer and use better terms.

The idea is that the collection_id you get is used in the data query
```
{
  collections: ["collection_id_1", "collection_id_2"],
  periods: ...,
  query: ...
}
```

I hope it's better now